FirebasePhoneAuthUI/FUIPhoneEntryViewController.h

```diff
@@ -40,25 +42,29 @@ __attribute__((deprecated("This is deprecated API and will be removed in a futur
 __attribute__((deprecated("This is deprecated API and will be removed in a future release."
                           "Please use initWithNibName:bundle:authUI:phoneNumber:")));
 
-/** @fn initWithAuthUI:phoneNumber:
+/** @fn initWithAuthUI:phoneNumber:countryCodes:
     @brief Convenience initializer.
     @param authUI The @c FUIAuth instance that manages this view controller.
     @param phoneNumber The phone number which is being verifying.
+    @param countryCodes Available country codes For the view controller to use.
```

Comment:
This isn't a part of your change, but nullable parameters should specify the behavior if `nil` is passed in (iirc in this case it'll just use the default value).

Reply:
Done.